Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -7777,6 +7777,37 @@ def string_to_context_key(string):
         raise Exception('The key is not a string: {}'.format(string))
 
 
+def response_to_context(reponse_obj):
+    """Recursively creates a data dictionary where all key starts with capital letters.
```

Comment:
It not only capitalizes the string it also splits words separated by underscore.
Let's add examples of input and output as you have in the `string_to_context_key`